doc/rados/operations/bluestore-migration.rst

```diff
@@ -136,37 +144,36 @@ to evacuate an entire host in order to use it as a spare, then the
 conversion can be done on a host-by-host basis with each stored copy of
 the data migrating only once.
 
-First, you need have empty host that has no data.  There are two ways to do this: either by starting with a new, empty host that isn't yet part of the cluster, or by offloading data from an existing host that in the cluster.
+First, you need an empty host that has no OSDs provisioned.  There are two
+ways to do this: either by starting with a new, empty host that isn't yet
+part of the cluster, or by offloading data from an existing host in the cluster.
 
 Use a new, empty host
 ^^^^^^^^^^^^^^^^^^^^^
 
 Ideally the host should have roughly the
-same capacity as other hosts you will be converting (although it
-doesn't strictly matter). ::
-
-  NEWHOST=<empty-host-name>
-
+same capacity as other hosts you will be converting.
 Add the host to the CRUSH hierarchy, but do not attach it to the root:
 
 .. prompt:: bash $
 
+   NEWHOST=<empty-host-name>
    ceph osd crush add-bucket $NEWHOST host
 
-Make sure the ceph packages are installed.
+Make sure that Ceph packages are installed on the new host.
 
 Use an existing host
 ^^^^^^^^^^^^^^^^^^^^
 
 If you would like to use an existing host
 that is already part of the cluster, and there is sufficient free
-space on that host so that all of its data can be migrated off,
-then you can instead do::
+space on that host so that all of its data can be migrated off to
+other cluster hosts, you can instead do::
 
-  OLDHOST=<existing-cluster-host-to-offload>
 
 .. prompt:: bash $ 
    
+   OLDHOST=<existing-cluster-host-to-offload>
```

Comment:
Is OLDHOST an environment variable (or something)?
This will cause a prompt to appear before it. If that's the intended formatting, great. If not, this should appear above the ".. prompt::" line.

Reply:
It's setting a shell variable to be referenced later.  I moved it because it seemed like it was in the wrong place.